utils/rotation_math/quaternion.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
```

Comment:
The quaternion lib was included to provide support of camera pose interpolation (easier to be done in quaternion rather than rotation matrix). Right now, since timestamp from realsense is not very accurate, interpolation is not supported yet. I will remove these lib for now.